indico/modules/rb/client/js/common/rooms/DailyAvailability.jsx

```diff
@@ -26,38 +26,37 @@ export default class DailyAvailability extends React.Component {
     value: [],
   };
 
-  handleTimesChange = ({startTime, endTime}, key) => {
+  handleTimesChange = ({startTime, endTime}, index) => {
     const {value, onChange} = this.props;
     onChange(
-      value.map(v => {
-        if (v.key === key) {
-          return {...v, startTime: serializeTime(startTime), endTime: serializeTime(endTime)};
-        } else {
-          return v;
-        }
-      })
+      value.map((v, vIndex) =>
+        vIndex === index
+          ? {...v, startTime: serializeTime(startTime), endTime: serializeTime(endTime)}
+          : v
+      )
     );
   };
 
-  handleRemoveTimes = key => {
+  handleRemoveTimes = index => {
     const {onChange, value} = this.props;
-    onChange([...value.filter(bH => bH.key !== key)]);
+    onChange(value.filter((_, i) => i !== index));
```

Comment:
I'd use `__` here since we often import lodash as `_`

Reply:
Agree, easier to distinguish.